### DISCUSSION TITLE
Error while running docker exec -it shopping-mongo /bin/bash cmd

### DISCUSSION THREAD

**Discussion by ravikumar401** [aspnetrun/run-aspnetcore-microservices#21] [Q&A]
Hi,

I am trying to run the below command, and it's giving the below error. do I need to Install the bash, if yes please let me know how to proceed further? I m using visual studio 2019 and windows 10 64 bit os.

**docker exec -it shopping-mongo /bin/bash**

**Error:** 

PS C:\Users\xxxx\source\repos\MicroSerivces\src> docker exec -it shopping-mongo /bin/bash
container 7434535004ee4f802561ca25f11a3ef8bb5b4f3a8a272c5c76874ddb77ddfe3a encountered an error during hcsshim::System::CreateProcess: failure in a Windows system call: The system cannot find the file specified. (0x2)
                                                                                       [Event Detail:  Provider: 00000000-0000-0000-0000-000000000000]
                    [Event Detail:  Provider: 00000000-0000-0000-0000-000000000000]
                                                                                   [Event Detail: onecore\vm\compute\management\orchestration\vmhostedcontainer\processmanagement.cpp(173)\vmcomputeagent.exe!00007FF6C6749F4B: (caller: 00007FF6C66FE13A) Exception(12) tid(390) 80070002 The system cannot find the file specified.
    CallContext:[\Bridge_ProcessMessage\VmHostedContainer_ExecuteProcess]
                                                                           Provider: 00000000-0000-0000-0000-000000000000] extra info: {"CommandLine":"/bin/bash","WorkingDirectory":"/","Environment":{"MONGO_DOWNLOAD_SHA256":"ede50e8f8d8c9d23a8ca2cc1c96cdb9bcc1f617930e8bd1d46f21d95d0b555f8","MONGO_DOWNLOAD_URL":"https://fastdl.mongodb.org/windows/mongodb-windows-x86_64-4.4.6-signed.msi","MONGO_VERSION":"4.4.6"},"EmulateConsole":true,"CreateStdInPipe":true,"CreateStdOutPipe":true,"ConsoleSize":[0,0]}

**Comment by mehmetozkaya** [comment]
Docker images should be linux image, could you please check your docker containers are linux based ? 
Also your Docker Desktop should be linux based configurations.

https://stackoverflow.com/questions/57453478/docker-run-gives-createprocess-failure-in-a-windows-system-call-the-system-ca
https://social.msdn.microsoft.com/Forums/en-US/193661e4-2979-420d-8610-41eb09754e88/docker-volumes-the-system-cannot-find-the-file-specified?forum=windowscontainers
